app/controllers/admin/routes.py

```diff
@@ -157,6 +159,54 @@ def rsvpLogDisplay(eventId):
     else:
         abort(403)
 
+@admin_bp.route('/event/<eventId>/renew', methods=['POST'])
+def renewEvent(eventId):
+    try: 
+        formData=request.form
+        try:
+            assert formData['timeStart'] < formData['timeEnd']
+        except AssertionError:
+            flash("End time must be after start time", 'warning')
+            return redirect(url_for('admin.eventDisplay', eventId = eventId))
+        
+        try:
+            if formData.get('dateEnd'):
+                assert formData['dateStart'] < formData['dateEnd']
+        except AssertionError:
+            flash("End date must be after start date", 'warning')
```

Comment:
Since the renew event modal does not have an end date, there is no instance where the end date would be before the start date. So this try/except might not be necessary.

Reply:
I would like to keep this in here because if the start date ever ends up after the end date for whatever reason, this would catch that error.